docker-compose.test.debug.yaml

```diff
@@ -1,3 +1,5 @@
+
+version: '3.8'
```

Comment:
Shouldn't need to add these tags anymore as they're now defunct in newer versions of docker compose.